spec/features/requests/accept_reject_request_spec.rb

```diff
@@ -36,17 +36,17 @@
 
       context 'with a filled in rejection information field' do
         before do
-          page.fill_in 'request[rejection_information]', with: 'Info'
+          page.fill_in 'request[rejection_information]', with: 'A rejection information'
           click_button('rejectButton')
         end
 
         it 'saves the rejection information' do
           request.reload
-          expect(request.rejection_information).to eq('Info')
+          expect(request.rejection_information).to eq('A rejection information')
         end
 
-        it 'shows rejection information' do
-          expect(page).to have_text('Info')
```

Comment:
We should (re)add a test that the rejection information is actually displayed to the user